rpc/block_svc_routines.c

```diff
@@ -210,6 +209,66 @@ mapJsonFlagToJsonCstring(int jsonflag)
   }
 }
 
+void
+log_command(char *cmd)
+{
+  FILE *fp;
+  char timestamp[GB_TIME_STRING_BUFLEN] = {0};
+
+
+  fp=fopen(gbConf.cmdhistoryLogFile,"a");
+  if (fp==NULL){
+    fprintf(stderr,"Error opening log file: %s\n"
+            "Logging to stderr.\n",strerror(errno));
+    fp = stderr;
+  }
+
+  logTimeNow(timestamp, GB_TIME_STRING_BUFLEN);
+  fprintf(fp, "[%s] %s\n", timestamp, cmd);
+  if (fp != stderr)
+    fclose(fp);
+}
+
+
+void
+collectRequestLogs(int errCode, const char *message, int json)
+{
+  FILE *fp;
+  char *p;
+  char timestamp[GB_TIME_STRING_BUFLEN] = {0};
+
+
+  fp = fopen(gbConf.cmdhistoryLogFile,"a");
+  if (fp == NULL) {
+    fprintf(stderr,"Error opening log file: %s\n"
+            "Logging to stderr.\n",strerror(errno));
+    fp = stderr;
+  }
+
+  logTimeNow(timestamp, GB_TIME_STRING_BUFLEN);
```

Comment:
Why do you need this function ?
Just print LOG("bla bla bla", reply->out);
post those block#OPT#CliFormatResponse() functions.